fulp_modules/lisa/joel_gun.dm

```diff
@@ -0,0 +1,120 @@
+/obj/item/gun/ballistic/revolver/joel
+	icon = 'icons/obj/guns/ballistic.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi' /// We probably don't want to change these...
+	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
+	load_sound = 'fulp_modules/lisa/Sounds/gunload.ogg'
+	eject_sound = 'fulp_modules/lisa/Sounds/empty.ogg'
+	icon_state = "revolver"
+	name = "\improper Bolt Action pistol"
+	desc = "The most powerful handgun in Olathe. It's best not to waste the only bullet."
+	mag_type = /obj/item/ammo_box/magazine/internal/cylinder/c22
+	var/used_ability = FALSE
+
+/// To prevent ability spam.
+/obj/item/gun/ballistic/revolver/joel/proc/clear_cooldown()
+	used_ability = FALSE
+
+/// Load It In
+/obj/item/gun/ballistic/revolver/joel/attackby(obj/item/A, mob/user, params)
+	..()
+	if(!istype(A, /obj/item/ammo_casing))
```

Comment:
what if this isn't the type of ammo casing that the gun normally uses? can you just spam-use a 10mm ammo casing or whatever on this gun to rapidly heal your brute damage?